client/src/components/game/lab2/LandingPage/mainInstructions.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Circle from '../components/circle';
+import './landingpage.css';
+
+const MainInstructions = () => {
+  return (
+    <div className='mainInstructionsContainer'>
+        {/*
+      <p className='instructionTitle'>
+        Main instructions for Color Clicker:
+      </p>
+        */}
+      <p className='mainInstructionList'>
+        You are about to play a game involving three colored circles, the same
+        size as this one:
+      </p>
+      <div className='center'>
+        <Circle color={'blue'} clickable={false}/>
+      </div>
+      <ul>
+        <li className='mainInstructionsItem'>
+          You will need to <strong>click the circle</strong> in the center of the screen.
+        </li>
+        <li className='mainInstructionsItem'>
+          The circle will be 1 of 3 colors. 
+        </li>
+        <li className='mainInstructionsItem'>
+          The color you need to click will appear in the <strong>bottom left</strong> corner
+          of the screen.
+        </li>
+        <li className='mainInstructionsItem'>
+          The colors you should <strong>avoid</strong> clicking will appear 
+          in the <strong>bottom right</strong> of the screen.
+        </li>
+        <li className='mainInstructionsItem'>
+          You will gain points based on if you clicked the <strong>correct or incorrect</strong> circle 
```

Comment:
The way this is stated sounds like you'll get points if you click on either the correct or incorrect circle. Maybe consider rewording for clarity? Like stating that you'll gain points from clicking on the correct circle and lose points from clicking on the incorrect circle